app/controllers/hyrax/admin/appearances_controller.rb

```diff
@@ -9,7 +9,6 @@ class AppearancesController < ApplicationController
       def show
         add_breadcrumb t(:'hyrax.controls.home'), root_path
         add_breadcrumb t(:'hyrax.dashboard.breadcrumbs.admin'), hyrax.dashboard_path
-        add_breadcrumb t(:'hyrax.admin.sidebar.configuration'), '#'
```

Comment:
These links are used inconsistently, so I removed this one since it's adjacent to work in this PR.